Fix legend spacing (HDS-2948) (#1856)
hashicorp/design-system: .changeset/sharp-phones-agree.md

---
"@hashicorp/design-system-components": patch
---

> `Form::Legend` - change spacing from 4px to 8px

Author: How should that be worded?

> `Form::Legend` - change spacing from 4px to 8px

Author: Updated it. Let me know if you have a better idea.

> `Form::Legend` & consuming components - changed spacing from 4px to 8px

Reviewer: We have to be explicit of which components have been impacted (see my other comment). We can't assume a consumer knows which are the "consuming components" of the `Form::Group`.

> `Form::Legend` & consuming components - changed spacing from 4px to 8px

Reviewer 2: Maybe this:

```suggestion
`Form::Legend` as consumed by `Checkbox::Group`, `Radio::Group`, and `RadioCard::Group` - changed spacing from 4px to 8px
```

I _think_ that captures all of the components this impacts, is that correct?

> `Form::Legend` as consumed by `Checkbox::Group`, `Radio::Group`, and `RadioCard::Group` - changed spacing from 4px to 8px

Reviewer: ```suggestion
`Form::Fieldset` as consumed by `Form::Checkbox::Group`, `Form::Radio::Group`, `Form::RadioCard::Group`, and `Form::Toggle::Group` - changed spacing from 4px to 8px
```

> `Form::Fieldset` as consumed by `Form::Checkbox::Group`, `Form::Radio::Group`, `Form::RadioCard::Group`, and `Form::Toggle::Group` - changed spacing from 4px to 8px

Reviewer: ```suggestion
`Form::Fieldset` as consumed by `Form::Checkbox::Group`, `Form::Radio::Group`, `Form::RadioCard::Group`, and `Form::Toggle::Group` - changed spacing between `legend` and content from 4px to 8px
```
